articles/native-platforms/android/02-custom-login.md

```diff
@@ -24,16 +24,16 @@ This tutorial and seed project have been tested with the following:
 <pre><code>a0${account.clientId}://\*.auth0.com/authorize</pre></code>
 </div>
 
-### 1. Add the Auth0 Java dependency
```

Comment:
Please change this document's title from `title: Auth0 Android Quickstarts - Custom Login` to `title: Custom Login`